lib/dictBuilder/cover.c

```diff
@@ -929,6 +929,65 @@ void COVER_best_finish(COVER_best_t *best, size_t compressedSize,
   }
 }
 
+static void COVER_selectDict(void* dict, size_t* dictBufferCapacity, const void* customDictContent, size_t dictContentSize,
+        const void* samplesBuffer, const size_t* samplesSizes, size_t nbTrainSamples, size_t nbSamples, ZDICT_cover_params_t params,
+        size_t* offsets, size_t* totalCompressedSize){
+
+  size_t largestDict = 0;
+  size_t largestCompressed = 0;
+  double defaultRegression = 1.01;
```

Comment:
Let's add the `autoShrinkDict` and `autoShrinkDictMaxRegression` parameters to COVER and FASTCOVER here https://github.com/facebook/zstd/blob/dev/lib/dictBuilder/zdict.h#L96, and use those in this function.

The regression should be an `unsigned` percent, instead of a `double`.